odoo/addons/base/models/res_partner.py

```diff
@@ -225,8 +225,7 @@ def _split_street_with_params(self, street_raw, street_format):
     company_name = fields.Char('Company Name')
 
     # image: all image fields are base64 encoded and PIL-supported
-    image = fields.Binary("Image",
-        help="This field holds the image used as avatar for this contact, limited to 1024x1024px",)
+    image = fields.Binary("Image")
```

Comment:
Why remove the help  message, if it is outdated, please rewrite it

Reply:
They were removed on Aline's request, the tooltips were displayed above the zoom and it was not pleasant to the eyes